src/kanata/mod.rs

```diff
@@ -1236,6 +1265,27 @@ impl Kanata {
                             }
                             pbtn
                         }
+                        CustomAction::MoveMouseSpeed {
+                            speed, /* todo: use this somehow */
+                            ..
+                        } => {
+                            log::debug!("resetting move_mouse_speed");
+                            // Revert the intervals to their old values
+                            if let Some(mmsh) = &mut self.move_mouse_state_horizontal {
```

Comment:
I'm pretty sure it's still not correct. The only difference - compared to before - is that the interval will be increasing instead decreasing with each subsequent use of mousemove-speed. Well, at least it won't panic because of underflow.

Reply:
One solution to fix this would be to have and additional `initial_interval` field for `MoveMouse`/`MoveMouseAccel` and change `interval` based on value of `initial_interval` and the list of active modifiers (suggested in the comment below this thread)

Reply:
> One solution to fix this would be to have and additional initial_interval field for MoveMouse/MoveMouseAccel and change interval based on value of initial_interval ...

Yes, this is probably how it should be. I'll implement these tomorrow.